docs/openid.md

```diff
@@ -166,6 +166,77 @@ oidc_providers:
         localpart_template: "{{ user.preferred_username }}"
         display_name_template: "{{ user.name }}"
 ```
+
+### [Authelia][authelia]
+
+[Authelia][authelia] is an open-source authentication and authorization server. Authelia supports the OIDC OP role currently in beta - you can follow the progress [here](https://www.authelia.com/docs/configuration/identity-providers/oidc.html#beta). While this setup is working, it is so far not advised to be run in production. In order to use this, you need to use the image authelia/authelia:feat-oidc-userinfo-endpoint or newer, since the neccesary endpoints are not included in the current release v.4.29.4.
+
+1. Follow the [Getting Started Guide](https://www.authelia.com/docs/getting-started.html) to install Authelia and configure the authentication backend.
+
+2. Create a private key with ```authelia rsa generate --dir /config```.
+
+3. Add the following section to the Authelia configuration.yaml.  
```

Comment:
I'm not sure if it makes sense to include a full configuration in here. Maybe we can point to some docs from Authelia instead and note what to change?